iModelCore/GeomLibs/geom/test/PolyfaceTest/t_ClipPlaneSet.cpp

```diff
@@ -2599,6 +2599,62 @@ TEST(ClipPlaneSet,ClipOpenBoundaryOutsideClipper)
     Check::ClearGeometry("ClipPlaneSet.ClipOpenBoundaryOutsideClipper");
     }
 
+/*---------------------------------------------------------------------------------**//**
+* @bsimethod
++---------------+---------------+---------------+---------------+---------------+------*/
+TEST(ClipPlaneSet,ClipOpenBoundaryWith2Parts)

```

Comment:
Test name could be more descriptive. This is a polygon configuration sometimes called a "split washer", i.e., an outer loop connected to an inner loop by a bridge edge that is traversed twice in opposite directions.

Reply:
I prefer current name over "split washer"
I googled it and it does not show what I expect to see. I think the name is clear after code readers visualize the curve.
So thanks for your suggestion but I prefer to keep the name as it is (ClipOpenBoundaryWith2Parts). Please note that the next test is ClipParityRegionWith2Parts.